tasmota/tasmota_xsns_sensor/xsns_18_pms5003.ino

```diff
@@ -205,28 +205,26 @@ bool ZH03ReadDataPassive() // process the passive mode response of the ZH03x sen
   AddLogBuffer(LOG_LEVEL_DEBUG_MORE, buffer, 9);
  
   uint8_t sum = 0;
-  for (uint32_t i = 1; i < 7; i++) {
+  for (uint32_t i = 1; i < 8; i++) {
     sum += buffer[i];
   }
-  sum=(~sum)+1;
+  sum=(~(sum&255))+1;
```

Comment:
Since sum is uint8_t, masking it with 0xff does not make sense.

Reply:
You are right, I'll change it